product_docs/docs/pgd/5/reference/routing.mdx

```diff
@@ -38,6 +38,16 @@ bdr.alter_proxy_option(proxy_name text, config_key text, config_value text);
 -   `config_key` &mdash; Key of the option in the proxy to be changed.
 -   `config_value` &mdash; New value to be set for the given key.
 
+The proxy options that can be changed using this function are:
+- `listen_address` &mdash; Address for the proxy to listen on.
+- `listen_port` &mdash; Port for the proxy to listen on.
+- `max_client_conn` &mdash; Maximum number of connections for the proxy to accept. Default is '32767'.
+- `max_server_conn` &mdash; Maximum number of connections the proxy can make to the Postgres node. Default is '32767'.
+- `server_conn_timeout` &mdash; Integer: Connection timeout for server connections. Default is '2' (seconds).
+- `server_conn_keepalive` &mdash; Keepalive interval for server connections. Default is '10' (seconds).
+- `consensus_grace_period` &mdash; Duration for which proxy continues to route even upon loss of a Raft leader. If set to 0s, proxy stops routing immediately. Default is '6' (seconds).
```

Comment:
Default is kind of `6s` for local proxies and `12s` for global proxies. 

Actually, the value depends on 
- `bdr.raft_response_timeout`, 
- `bdr.raft_global_election_timeout`, and  
- `bdr.raft_group_election_timeout`.  

Exact logic is given below but this could change in future versions to we should be discreet here I guess.

```sql
  coalesce(
    nullif(current_setting('bdr.raft_response_timeout', true), '-1')::interval,
    case when h.node_group_type = 'global' then
      nullif(current_setting('bdr.raft_global_election_timeout', true)::interval, '-1') * 2
    else
      nullif(current_setting('bdr.raft_group_election_timeout', true)::interval, '-1') * 2
    end,
    '0s'::interval
  ) consensus_grace_period
```